spyder/utils/conda.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Conda/anaconda utilities."""
+
+# Standard library imports
+import os
+import sys
+
+
+def add_quotes(path):
+    """Return quotes if needed for spaces on path."""
+    quotes = '"' if ' ' in path and '"' not in path else ''
+    return '{quotes}{path}{quotes}'.format(quotes=quotes, path=path)
+
+
+def is_conda():
+    """Check if it is a conda environment, if it is run activation script."""
+    return os.path.exists(os.path.join(sys.prefix, 'conda-meta'))
+
+
+def get_conda_root_prefix(quote=False):
+    """
+    Return conda prefix from sys.prefix.
+
+    If `quote` is True, then quotes are added if spaces are found in the path.
+    """
+    env_key = '{0}envs{0}'.format(os.sep)
+    if sys.prefix.rfind(env_key) != -1:
+        root_prefix = sys.prefix.split(env_key)[0]

```

Comment:
I think this wouldn't work if Spyder is installed in a different environment from the base env.

Reply:
That is why there is an else :-)

Reply:
if `/envs/` is not found (== -1) use sys.prefix otherwise it is a non base env.